src/Tribe/Tickets_View.php

```diff
@@ -475,6 +475,14 @@ public function inject_link_template() {
 
 		$file = Tribe__Tickets__Templates::get_template_hierarchy( 'tickets/orders-link.php' );
 
+		/**
+		 * @since TBD Attempt to load from old location to account for pre-existing theme overrides. If not found,
+		 *            go through the motions with the new location.
+		 */
+		if ( empty( $file) ) {
```

Comment:
:no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: c34b4b01ac8dd1bc9cbb090a8c11bb50</sub>